### ISSUE TITLE
Big oxygen tank doesn't fit in back slot

### ISSUE THREAD

**Issue by Jibbajabbafic** [unitystation/unitystation#1860]
# Description
You should be able to equip the oxygen tank in your back slot

This should also make sure that auto equipping (pressing E) can equip it into the back slot too.

## Category
BUG

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1. Grab big oxygen tank from O2 locker
2. Click back slot

**Also:**
1. Fill up pockets and suit storage with items
2. Take off backpack (by dragging into hands)
3. Hold oxygen tank
4. Press E
5. Notice it doesn't go into the back slot

**Comment by VictorVow** [comment]
Note in tg this doesn't work with every exosuit item. EVA suits can equip it but Scientist labcoats cannot. Other known exosuits which can equip large oxygen tanks to suit storage are hardsuits, bio suit and bomb suit.

**Comment by Jibbajabbafic** [comment]
This is for the backpack slot, I'll make a separate issue for suit storage.
Thanks for the info though, I wasn't sure which suits could hold the big tanks.

**Comment by Jibbajabbafic** [comment]
Issue made: #1877

**Comment by chairbender** [comment]
This should be easily fixable after #2245

**Comment by cvdva** [comment]
Unable to duplicate this issue. I think it's been solved by another user - it has the correct settings in the prefab.
<img width="337" alt="Screen Shot 2019-12-02 at 4 48 51 PM" src="https://user-images.githubusercontent.com/56093742/69998103-9ff69f00-1523-11ea-96f9-78bd1e64fbb5.png">
